twitter.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="twitter.css">
+    <title>Twitter</title>
+</head>
+<body>
+    <div class="main-body three-column-grid center-wrapper">
```

Comment:
Maybe you could use comments <!- -> to write which section is what, so that it is easier to read the code